src/app/Swasti/SponsorCompany.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import image1 from "./Assets/1.png"
+import image2 from "./Assets/2.png"
+import image3 from "./Assets/3.png"
+import image4 from "./Assets/4.png"
+import image5 from "./Assets/5.png"
+import image6 from "./Assets/6.png"
+import "./style.css"
+import Image from 'next/image'
+const SponsorCompany = () => {
+  const images = [image1, image2, image3]
+  return (
+    <>
+    <section className='main-section'>
+    
+    {
+      images.map((image, index) => {
+        return (
+          <div className='Big-dot'>
```

Comment:
add a key prop in the outermost div, You may use index as the key


***Build Logs***
```bash
Failed to compile.
6:14:59 PM: ./src/app/Swasti/SponsorCompany.jsx
6:14:59 PM: 19:11  Error: Missing key prop for element in iterator  react/jsx-key
6:14:59 PM: info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules

```